Extension/package.nls.json

```diff
@@ -182,6 +183,8 @@
     "c_cpp.configuration.exclusionPolicy.checkFilesAndFolders.description": "The exclusion filters will be evaluated against every file and folder encountered.",
     "c_cpp.configuration.preferredPathSeparator.markdownDescription": { "message": "The character used as a path separator for `#include` auto-completion results.", "comment": [ "Markdown text between `` should not be translated or localized (they represent literal text) and the capitalization, spacing, and punctuation (including the ``) should not be altered." ] },
     "c_cpp.configuration.simplifyStructuredComments.markdownDescription": { "message": "If `true`, tooltips of hover and auto-complete will only display certain labels of structured comments. Otherwise, all comments are displayed.", "comment": [ "Markdown text between `` should not be translated or localized (they represent literal text) and the capitalization, spacing, and punctuation (including the ``) should not be altered." ] },
+    "c_cpp.configuration.autocompleteDoxygenComment.markdownDescription": { "message": "Whether to automatically insert the doxygen comment after typing the chosen comment style", "comment": [ "Markdown text between `` should not be translated or localized (they represent literal text) and the capitalization, spacing, and punctuation (including the ``) should not be altered." ] },
```

Comment:
I think we're supposed to end the descriptions with a  period.

Reply:
Add "Controls" to the beginning of "whether to automatically insert the Doxygen comment after typing the chosen comment style." This is to follow the existing descriptions wording style.